website/views/shared/modals/members.jade

```diff
@@ -46,9 +46,12 @@ script(type='text/ng-template', id='modals/private-message.html')
   .modal-header
     h4=env.t('pmHeading', {name: "{{profile.profile.name}}"})
   .modal-body
-    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',ng-model='_message')
+    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',placeholder=env.t('needsTextPlaceholder'),ng-model='_message')
```

Comment:
It's currently possible to send the message by pressing `meta-enter`, which is `ctrl+enter` on a pc and `cmd+enter` on a mac. You may need to edit the `sendPrivateMessage` function to not send if `_message` is blank.